dbms/src/TableFunctions/parseColumnsListForTableFunction.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <Core/Block.h>
+
+
+namespace DB
+{
+
+void parseColumnsList(const std::string & structure, Block & sample_block, const Context & context);
```

Comment:
Comment needed.
Better to return Block by value.

Reply:
There is comment in later commit